SwanCustomEnvironments/swancustomenvironments/templates/customenvs.html

```diff
@@ -229,7 +229,7 @@ <h2 id="modal-title" style="text-align: left;"></h2>
                     else {
                         outputBox.innerHTML += message_line;
                         outputBox.scrollTop = outputBox.scrollHeight;
-                        if (message_line.includes('ERROR')) {
+                        if (message_line.toUpperCase().includes('ERROR')) {
```

Comment:
Why do you need the uppercase? Isn't ERROR in lowercase your signal for when something is wrong? Is it to catch other errors that you do  not signal?